OrbitGl/Batcher.cpp

```diff
@@ -67,6 +68,20 @@ void Batcher::AddShadedBox(Vec2 pos, Vec2 size, float z, Color color,
   AddBox(box, colors, picking_type, user_data);
 }
 
+void Batcher::AddTriangle(const Triangle& triangle, Color color,
```

Comment:
Don't know how much of a difference it really makes but since this might be mildly preformance critical consider using a rvalue reference for the primitive. Same for Box and Line. Maybe this is rather a follow up optimization.

Reply:
Ahh, forget it. It is passed by reference down to the Block and there you need a copy anyway.